packages/flutter_localizations/lib/src/l10n/material_es.arb

```diff
@@ -9,6 +9,8 @@
   "previousMonthTooltip": "Mes anterior",
   "nextPageTooltip": "Página siguiente",
   "previousPageTooltip": "Página anterior",
+  "firstPageTooltip": "Página primera",
+  "lastPageTooltip": "Página última",
```

Comment:
Shouldn't these (and all the other es variants) be the English placeholders?

Reply:
Edit: I just noticed you said you added the Spanish equivalents for them. Google's internal translation console will eventually provide these translations. 

We do not take community translations since 1) they will be overridden anyway by Google's translation system and 2) the engineering team has no way to check if they're correct and is not spam (for example, in the worst case, someone puts in profanity or very incorrect translations and we have no idea).

Reply:
Sounds good, I'll roll them back to English. That's actually good to know that I won't have to worry about that in the future

Reply:
Done